mapscript/mapscript.i

```diff
@@ -33,6 +33,8 @@
 %begin %{
 #ifdef _MSC_VER
 #define SWIG_PYTHON_INTERPRETER_NO_DEBUG
+#else
+#define _GNU_SOURCE 1
```

Comment:
Just `#define _GNU_SOURCE` is enough, it doesn't need a value.

Reply:
I did at first but it throws several 'redefined' warnings.  Giving it a value prevents the warnings.

Reply:
Then you probably shouldn't define it unconditionally, e.g.:
```C
#ifdef _MSC_VER
#define SWIG_PYTHON_INTERPRETER_NO_DEBUG
#elif !defined(_GNU_SOURCE)
#define _GNU_SOURCE
#endif
```

Reply:
I had tried that earlier as well.  I tried again, still gives warning "/../main/php_config.h:2535:0: warning: _GNU_SOURCE" redefined".

Reply:
With all my testing I found the best way was the pass it a value, to remove the redefinition warnings in PHP source.

Reply:
Alright, then just leave it as-is.

Reply:
But I'm open to other ways.  Let me know.

Reply:
Looking at [codesearch.debian.net](https://codesearch.debian.net/search?q=%23define+_GNU_SOURCE&literal=0&perpkg=1&page=4) there a quite a number of occurences with value, so either seems acceptable.

Reply:
ok thanks